webrecorder/browsertrix
# QA review Replay embed improvements

Resolves https://github.com/webrecorder/browsertrix/issues/1758



### Changes

- Adds button to refresh current QA review Replay
- Shows loading indicator when Replay window is loading
- Prevents clicking links within Replay
- Fixes document scrollbar gutter
- Adds scroll snap to QA review

### Manual testing

1. Log in as crawler
2. Navigate to crawl QA review
3. Click "Replay" tab. Verify Replay window loads as expected
4. Click link within Replay. Verify click is prevented and message is shown
5. Click page in page list. Verify loading indicator and page content shows as expected
6. Change QA run ID. Verify loading indicator and page content shows as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| QA Review - Replay | <img width="402" alt="Screenshot 2024-04-30 at 6 31 59 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/9fec3f65-c844-42b2-91a5-e7d83a95366a"> |
| QA Review - Replay (new page loading) | <img width="911" alt="Screenshot 2024-04-30 at 6 30 43 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/79acc544-21b5-4e75-8ca1-d7fe358ce760"> |
| QA Review - Replay (click prevented) | <img width="884" alt="Screenshot 2024-04-30 at 6 30 56 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/ad2027e5-4240-4bf8-9224-fb8b961134c8"> |
